### PR TITLE
Change history blog titles

### PR DESCRIPTION
**Pull request by septemberfd**
### What is changed, added or deleted? (Required)

Modify the URLs of 24 history blog articles to make them more searchable on Google. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

@ran-huang

### CODE REVIEW

**Comment by ran-huang** [comment]
You can ignore the `manual line breaks` error for now. It's a known bug (https://github.com/pingcap/docs/issues/6490). I'll fix it soon.

File: announcement.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ date: 2017-10-17
 summary: TiDB is compatible with MySQL, strong consistent and highly available.
```
Comment (ran-huang) [inline announcement.md:4]: The new filename is too general. I would use something like `announcing-tidb-1.0-release.md`. See https://pingcap.feishu.cn/docs/doccnIShlpKetyVrDl5tOrSduef#0ArnXY for detailed URL tips.

File: futuresandgrpc.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ author: ['Siddon Tang']
 date: 2017-09-12
 summary: This is the speech Siddon Tang gave at Bay Area Rust Meetup August 2017.
```
Comment (ran-huang) [inline futuresandgrpc.md:5]: Ditto. `futures-and-grpc-in-rust.md`

File: multi-raft.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: The Design and Implementation of Multi-raft
 date: 2017-08-15
```
Comment (ran-huang) [inline multi-raft.md:3]: ditto

File: mvcc-in-tikv.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ author: ['Ivan Yang']
 date: 2016-11-17
```
Comment (ran-huang) [inline mvcc-in-tikv.md:4]: `multi-version-concurrency-control-in-tikv.md`

File: nextcon.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ author: ['Li SHEN']
 date: 2017-10-10
```
Comment (ran-huang) [inline nextcon.md:4]: Ditto.

**Review by ran-huang — COMMENTED** [review]
Please apply my comments to all 24 articles.

File: Comparison-between-MySQL-and-TiDB-with-tens-of-millions-of-data-per-day.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ title: Migration from MySQL to TiDB to handle tens of millions of rows of data p
 date: 2017-05-22
```
[the remark this answers is not in the document]
Comment (septemberfd) [reply Comparison-between-MySQL-and-TiDB-with-tens-of-millions-of-data-per-day.md:3]: done

File: announcement.md (hunk 1 above)
Comment (septemberfd) [reply announcement.md:4]: done

File: futuresandgrpc.md (hunk 2 above)
Comment (septemberfd) [reply futuresandgrpc.md:5]: done

File: multi-raft.md (hunk 3 above)
Comment (septemberfd) [reply multi-raft.md:3]: done

File: mvcc-in-tikv.md (hunk 4 above)
Comment (septemberfd) [reply mvcc-in-tikv.md:4]: done

File: nextcon.md (hunk 5 above)
Comment (septemberfd) [reply nextcon.md:4]: done

**Review by ran-huang — APPROVED** [review]

**Pull request merged**